build-logic/publication/src/main/kotlin/convention.publish-base.gradle.kts

```diff
@@ -11,16 +9,6 @@ version = providers.gradleProperty("projectVersion")
     .forUseAtConfigurationTime()
     .get()
 
-tasks.withType<Jar> {
-    manifest {
-        attributes(
-            mapOf(
-                Attributes.Name.IMPLEMENTATION_VERSION.toString() to project.version
-            )
-        )
-    }
-}
```

Comment:
Moved to conventions for libraries. 
Build metadata will be available for any library, even not intended for publishing.